src/redux/Destinations/destinations.js

```diff
@@ -1,19 +1,28 @@
-import { getDestinations } from '../../helpers/getDestinationData';
+const GET_DESTINATIONS = 'GET_DESTINATIONS';
 
```

Comment:
Please follow the duck pattern rule:
MUST have action types in the form npm-module-or-app/reducer/ACTION_TYPE - [link](https://github.com/erikras/ducks-modular-redux#rules)

This will allow you to use shorter names for const variables. So, instead of GET_DESTINATIONS you can use just GET. It is part of destinations reducer so you know that it relates to destinations.